### ISSUE TITLE
Percent complete is 99% for tW ru_rlob-2pe_book but it is all checked

### ISSUE THREAD

**Issue by birchamp** [unfoldingWord/translationCore#7461]
This project https://git.door43.org/ru_gl/ru_rlob_2pe_tcore_checked_tW-checked_tN-aligned shows 99% complete even though it is entirely checked.

**Comment by birchamp** [comment]
@elsylambert at some point when you're testing tC can you test this issue?

**Comment by elsylambert** [comment]
@birchamp Which tC version are you seeing this?

**Comment by elsylambert** [comment]
Tested this project in translationCore 3.4.0 (146a9f1) .
After importing the project, tW shows 99% completion, when no GL was selected. After choosing English-en-D43 catalog as GL, the tools card shows 100% completion.
<img width="1454" alt="Screenshot 2022-12-07 at 9 48 36 AM" src="https://user-images.githubusercontent.com/51806180/206210978-1feba6a4-0276-4a76-82a4-578f25d358e2.png">

- Download ru content, ru_gl, tW resources from content updater.
- Select ru_gl as GL for tW card, the completion goes down to 99%.
- 
<img width="1454" alt="Screenshot 2022-12-07 at 9 48 25 AM" src="https://user-images.githubusercontent.com/51806180/206211429-7ae3373d-fb1a-4f90-a505-48951c19b032.png">

- Uncheck the "Key words" checkbox, the completion goes to 100%
-
<img width="1454" alt="Screenshot 2022-12-07 at 9 48 16 AM" src="https://user-images.githubusercontent.com/51806180/206211681-adbd61a1-2229-4bf1-b4e6-0f301c1c3256.png">

**Comment by elsylambert** [comment]
Note that all the checks in tW are completed. @PhotoNomad0

**Comment by PhotoNomad0** [comment]
@elsylambert How did you download this app? tCore is throwing an error for me that the url is not valid.

**Comment by elsylambert** [comment]
@PhotoNomad0 I did online import using the URL  https://git.door43.org/ru_gl/ru_rlob_2pe_tcore_checked_tW-checked_tN-aligned last week.

**Comment by PhotoNomad0** [comment]
@elsylambert Oops, I'm using QA server!

**Comment by PhotoNomad0** [comment]
@elsylambert I'm wondering if this is because they have 4 tw checks that are marked as no selection is needed?

**Comment by PhotoNomad0** [comment]
@elsylambert Yes, making selections for those four checks eventually gets to 100%.  Will have to check the algorithm to see if makes use of no checks needed.

**Comment by PhotoNomad0** [comment]
@elsylambert @birchamp - It looks like there is a check that is failing:

![Screenshot 2022-12-12 at 12 07 19 PM](https://user-images.githubusercontent.com/14238574/207108781-c2606aa7-8449-41b0-aa52-d46f1a554ca5.png)

It looks like the groupId should have been "righteous" instead of "righteousness"?  Not sure why it is not displayed though.

Here is the check in https://git.door43.org/ru_gl/ru_twl/src/branch/master/twl_2PE.tsv on line186.  The path is rc://*/tw/dict/bible/kt/righteousness, but in https://git.door43.org/ru_gl/ru_tw/src/branch/master/bible/kt there is no article  righteousness.md, but there is one for righteous.md.  It's looking like a bug that tW is not showing the checks when the article is missing. Continuing to look...

**Comment by PhotoNomad0** [comment]
Notes:
- see generateMenuData() in ./tc-ui-toolkit/src/GroupedMenu/utils.js - it probably should handle entries from index (groupsIndex) whose id is not in  data (groupsData).  Or maybe tCore should create categories for check items not defined.
